Backpacks.cs

```diff
@@ -666,6 +708,29 @@ public ushort BackpackSize
 
             [JsonProperty("Blacklisted Items (Item Shortnames)")]
             public HashSet<string> BlacklistedItems;
+
+            [JsonProperty(PropertyName = "GUI button")]
+            public GUIButton GUI = new GUIButton();
+
+            public class GUIButton
+            {
+                [JsonProperty(PropertyName = "Enabled?")]
+                public bool Enabled = true;
```

Comment:
I suggest we do a permission instead of a config option since it's more flexible and allows admins to safely test it out before adding for others. Admins like the peace of mind knowing they do a gradual rollout like this without having to test on another server (though they should test on another server if they know what's good for them).